HTWDD Today/TodayViewController.swift

```diff
@@ -137,6 +138,20 @@ class TodayViewController: ViewController {
 		}
 	}
 	
+    // MARK: - UIGestureRecognizer
+    
+    fileprivate func addGestureRecognizer() {
+        
+        let gestureRecognizer = UITapGestureRecognizer(target: self, action: #selector(self.handleTap(gestureRecognizer:)))
+        gestureRecognizer.delegate = self
+        self.containerView.addGestureRecognizer(gestureRecognizer)
+    }
+    
+    // MARK: - URL
+    
+    @objc func handleTap(gestureRecognizer: UIGestureRecognizer) {
+        extensionContext?.open(URL(string:"htwdd://schedule")!, completionHandler: nil)
```

Comment:
Where do you think should I put the global constant for the url scheme, the enum with the suffix and the method? 
Should they be part of the AppCoordinator?

Reply:
You could make use of the fancy `HTWNameSpace` and create an extension of it ;) This would look like this: `extension HTWNamespace where Base == URL { }`.

In there you can create your properties and functions and then access them via `URL.htw.someProperty`.